### DISCUSSION TITLE
Filmes e séries

### DISCUSSION THREAD

**Discussion by onhernandes** [frontendbr/forum#789] [Dicas]
Galera, no começo do ano fiz uma [lista com alguns filmes que envolvem tecnologia de alguma forma](https://github.com/onhernandes/awesome-movies), mas não atualizo desde Janeiro e não divulguei por acreditar que ainda está incompleta. Pretendo reorganizar aquele README e deixar tabulado.

Não conheço muitos filmes ou séries que envolvem tecnologia, os que eu conheço eu coloquei na lista. Podem me dar indicações?

Também, se quiserem abrir uma issue diretamente no repositório com o nome do filme/série, seria ótimo!

[https://github.com/onhernandes/awesome-movies](https://github.com/onhernandes/awesome-movies)

**Comment by didiraja** [comment]
Impossível não cite Mr. Robot uma das melhores sobre tecnologia atualmente e pra mim, a mais verídica já feita.

**Comment by ghost** [comment]
isso é perda de tempo pois já existe o https://hackermovie.club/

**Comment by brunomosciatti** [comment]
Silicon Valley é a melhor de todas para nossa área =)

**Comment by mahenrique94** [comment]
Acho que Silicon Valley e Mr. Robot são as melhores atualmente.

**Comment by lucascdsilva** [comment]
Achei bem legal, 
Invasores- Nenhum Sistema Está à Salvo
https://www.youtube.com/watch?v=YQ-_BWFsL1o
